### PR TITLE
[10.3] [PR 2118] Reorder the admin nav items into alphabetical order

### PR DESCRIPTION
**Pull request by settermjd**
Backport of PR #2118

### CODE REVIEW

File: modules/admin_manual/nav.adoc [hunk 1]
```diff
@@ -97,16 +91,22 @@
 **** xref:configuration/server/harden_server.adoc[Harden Server]
 **** xref:configuration/server/import_ssl_cert.adoc[Import SSL Cert]
 **** xref:configuration/server/index_php_less_urls.adoc[Index PHP Less URLs]
+**** xref:configuration/server/occ_command.adoc[OCC Command]
 **** xref:configuration/server/language_configuration.adoc[Language Configuration]
 **** xref:configuration/server/legal_settings_configuration.adoc[Legal Settings Configuration]
 **** xref:configuration/server/logging/logging_configuration.adoc[Logging Configuration]
 ***** xref:configuration/server/logging/request_tracing.adoc[Request Tracing]
+**** xref:configuration/server/reverse_proxy_configuration.adoc[Reverse Proxy Configuration]
 ***** xref:configuration/server/logging/providing_logs_and_config_files.adoc[Retrieve Log Files and Configuration Settings]
```
Comment (pako81) [inline modules/admin_manual/nav.adoc]: invert line 99 and 100?

Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Well spotted. Thanks.

Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Done!

File: modules/admin_manual/nav.adoc [hunk 2]
```diff
@@ -99,16 +93,22 @@
 **** xref:configuration/server/harden_server.adoc[Harden Server]
 **** xref:configuration/server/import_ssl_cert.adoc[Import SSL Cert]
 **** xref:configuration/server/index_php_less_urls.adoc[Index PHP Less URLs]
+**** xref:configuration/server/occ_command.adoc[OCC Command]
```
Comment (mmattel) [inline modules/admin_manual/nav.adoc]: why do you have "o" after "i" and not before "r" ?

File: modules/admin_manual/nav.adoc [hunk 3]
```diff
@@ -99,16 +93,22 @@
...
-**** xref:configuration/server/oc_server_tuning.adoc[Server Tuning]
-**** xref:configuration/server/occ_command.adoc[OCC Command]
 **** xref:configuration/server/reverse_proxy_configuration.adoc[Reverse Proxy Configuration]
-**** xref:configuration/server/security_setup_warnings.adoc[Security Setup Warnings]
+**** xref:configuration/server/security/index.adoc[Security]
+***** xref:configuration/server/security/brute_force_protection.adoc[Brute-Force Protection]
+***** xref:configuration/server/security/oauth2.adoc[OAuth2]
+***** xref:configuration/server/security/password_policy.adoc[Password policy]
+***** xref:configuration/server/security_setup_warnings.adoc[Security Setup Warnings]
+***** xref:configuration/server/security/hsmdaemon/index.adoc[The HSM (Hardware Security Module) Daemon]
```
Comment (mmattel) [inline modules/admin_manual/nav.adoc]: Wouldn´t it look better to write:
`The HSM (Hardware Security Module) Daemon`--> `HSM (Hardware Security Module) Daemon`
and reorder it accordingly?

File: modules/admin_manual/nav.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/nav.adoc]: I don't know what you're referring to.

File: modules/admin_manual/nav.adoc (hunk 3 above)
Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Might do. No harm in changing it.

Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Implemented.

File: modules/admin_manual/nav.adoc (hunk 2 above)
Comment (mmattel) [reply modules/admin_manual/nav.adoc]: (I)ndex PHP Less URL
(O)CC Command
(L)anguage Configuration
...
(R)everse Proxy Configuration

OCC Command should be right above Reverse Proxy Configuration and not below Index PHP Less URL

Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Thanks.

Comment (settermjd) [reply modules/admin_manual/nav.adoc]: Done.

**Pull request merged**